.circleci/config.yml

```diff
@@ -10,6 +10,15 @@ test_runner: &test_runner
   image: datadog/docker-library:dd_trace_py_1_1_0
   env:
     TOX_SKIP_DIST: True
+restore_cache_step: &restore_cache_step
```

Comment:
We should probably document the usage and motivation behind `Environment.CACHE_EXPIRE_HASH` so that it doesn't look like magic or require much digging in the future 😄

Reply:
Great idea. Will do right now! 👍

Reply:
@Kyle-Verhoog just pushed an update. Would you please take a look at it? Thanks